resources/assets/js/components/Application/ExperienceModals/PersonalExperienceModal.tsx

```diff
@@ -280,7 +276,7 @@ export const newPersonalExperience = (
   description: "",
   is_shareable: false,
   is_active: false,
-  start_date: new Date(),
+  start_date: dayjs().format("YYYY-MM-DD"),
```

Comment:
It might be nice to abstract this away to a `newDateString()` helper function. I know its super simple, but I like not having to remember or repeat the format string in multiple places.